usr/share/rear/conf/default.conf

```diff
@@ -1028,6 +1028,10 @@ ISO_DEFAULT="boothd"
 # (e.g. when the device with the ISO is the first one in the BIOS boot order list).
 # The default ISO_RECOVER_MODE="" results the normal behaviour:
 ISO_RECOVER_MODE=""
+#
+# SYSLINUX timeout in 0.1 seconds units (e.g. 100 = 10 seconds) to automatically boot 
+# set this value to 0 to disable
+ISO_SYSLINUX_TIMEOUT="$(( $USER_INPUT_TIMEOUT * 10 ))"
```

Comment:
This won't work as intended because we read `default.conf` before the `site.conf` so that this only changes the default without changing the value if a user sets `USER_INPUT_TIMEOUT` in configuration.

Reply:
The same would also affect `GRUB2_TIMEOUT`:
```
# GRUB2_TIMEOUT
# The timeout in seconds to automatically boot GRUB2_DEFAULT_BOOT
# when GRUB2 is used as bootloader for the ReaR recovery system.
GRUB2_TIMEOUT="$USER_INPUT_TIMEOUT"
####
```

Reply:
"Dynamic defaults" is a new concept to me.

As far as I know (perhaps there are exceptions that I don't
remember right now) what we have are static defaults
which are set explicitly in default.conf and
if config variables are empty then ReaR does things on its own.